community/google-summer-of-code/reports/2017/lambda-Support-in-debugger-expression-evaluator.md

```diff
@@ -0,0 +1,135 @@
+---
```

Comment:
please lowercase the S in the file name (Support->support): `lambda-support-in-debugger-expression-evaluator.md`

Reply:
Done.